android/app/src/main/res/values/styles.xml

```diff
@@ -11,7 +11,7 @@
         <item name="android:background">@drawable/background_splash</item>
     </style>
     <style name="SplashScreen">
-    <item name="android:src">@mipmap/splash_beta</item>
+    <item name="android:src">@mipmap/splash</item>
```

Comment:
will the splash screen be changed in the bitrise during deployment proccess?

Reply:
yes